templates/taxi/car_detail.html

```diff
@@ -19,4 +19,15 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+  {% if request.user in car.drivers.all %}
+    <form method="post" action="{% url 'taxi:driver-delete' pk=car.id %}">
```

Comment:
The form action URL 'taxi:driver-delete' with 'pk=car.id' might not be correct for removing the current user as a driver from the car. Ensure that the URL corresponds to the correct view and that the view logic supports this action.